composer.json

```diff
@@ -23,8 +23,8 @@
 		}
 	],
 	"support" : {
-		"issues" : "http://code.google.com/p/php-sql-parser/issues/list",
-		"source" : "http://php-sql-parser.googlecode.com/svn/trunk/"
+		"issues" : "https://github.com/greenlion/PHP-SQL-Parser/issues",
+		"source" : "https://github.com/greenlion/PHP-SQL-Parser"
```

Comment:
this could be omitted entirely as Composer is able to guess them for Github repos